libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -892,7 +888,7 @@ def deploy_cortx_k8s_re_job(master_node_list: list, worker_node_list: list,
         return False, output['result']
 
     @staticmethod
-    def get_hctl_status(node_obj, pod_name: str) -> tuple:
+    def get_hctl_status(node_obj) -> tuple:
```

Comment:
Please make sure all occurrences are change for this function call.

Reply:
yes it taken care